agent_api/src/test/java/wrappers/MSSQLWrapperTest.java

```diff
@@ -40,6 +43,7 @@ public void tearDown() throws SQLException {
             connection.close();
         }
         Context.set(null);
+        StatisticsStore.clear();
```

Comment:
prob setUp is enough?

Reply:
we try to not affect other test cases as well, but in this case can probably remove